insights_content_template_renderer/models.py

```diff
@@ -0,0 +1,141 @@
+from typing import List, Dict
+import uuid
+
+from pydantic import BaseModel
+
+
+class Content(BaseModel):
+    plugin: dict
+    error_keys: dict
+    generic: str
+    summary: str
+    resolution: str
+    more_info: str
+    reason: str
+    HasReason: bool
+
+    class Config:
+        schema_extra = {
+            "example": {
```

Comment:
is it worth to  load this schema from a separate `json` file?

Reply:
updated PTAL